<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageListPage</class>
 <widget class="QWidget" name="MessageListPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newMessage">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Send a new message</string>
       </property>
       <property name="text">
        <string>New Msg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyMessage">
       <property name="toolTip">
        <string>Copy the currently selected message to the system clipboard</string>
       </property>
       <property name="text">
        <string>&amp;Copy Msg ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detailButton">
       <property name="toolTip">
        <string>Get message details</string>
       </property>
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="toolTip">
        <string>Refresh message list</string>
       </property>
       <property name="text">
        <string>&amp;Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="toolTip">
        <string>Export the data in the current tab to a file</string>
       </property>
       <property name="text">
        <string>&amp;Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syscoin.qrc"/>
 </resources>
 <connections/>
</ui>
